### PR TITLE
adding a nice mobile-friendly footer

### PR DESCRIPTION
**Pull request by tadler**
Translations aren't showing up*, but imagine there were some nice words here:

screen shot 2018-10-12 at 2 32 44 pm

*(Is there a way to fix that in this branch? I still have missing translations in other places so I've been assuming it's just environment-related.)

Pivotal: 
https://www.pivotaltracker.com/story/show/160263227

cc @MariaBianchi

### CODE REVIEW

File: resources/dictionary.edn [hunk 1]
```diff
@@ -492,7 +492,10 @@
    "이 웹사이트는 유권자와 투표지를 제출하기에 필요한 선거 정보를 연결해 주는 주 선거 관리 공무원과 민주주의 노력 사이에 파트너십인 투표 정보 프로젝트 (VIP) 발의입니다. 2008년에 시작하여, VIP는 공식적이고 최신 선거 정보를 제공하기 위해 주 및 지방 선거 관리 공무원과 함께 일합니다."}},
  :en
  {:footer
-  {:powered-by "Powered by "}
+  {:project-of "A project of "},
```
Comment (ddellacosta) [inline resources/dictionary.edn]: These should all be in one map like:

```clojure
{:footer
  {:project-of "A project of "
   :report-a-problem "Report a problem with the site"
   :privacy-policy "Privacy policy"
   :terms-of-service "Terms of service"}
...
```

As it is, I'm not entirely sure how this is going to get imported, but by default Clojure interprets a map structured this way as each individual map being its own key/value pair inside the outer map (this is also probably why the build is breaking? https://buildkite.com/the-voting-information-project/voting-info-tool-v2/builds/492#839a4e1a-d940-419a-bc0e-bdd24616e3d3/35-75).
```

Comment (tadler) [reply resources/dictionary.edn]: Done!

File: src/cljs/gttp2/views.cljs [hunk 2]
```diff
@@ -85,8 +85,20 @@
...
   [:section.footer
    [:div.footer-content
-    [:span (tr lang :footer/powered-by)]
-    [:a.footer-link {:href "http://democracy.works"} "Democracy Works"]]])
+    [:ul.footer-links.footer-links--primary
+      [:li.footer-link--item
+        [:a.footer-link {:href "https://www.votinginfoproject.org/"} "The Voting Information Project"]]
+      [:li.footer-link--item
+        [:span (tr lang :footer/project-of)]
+        [:a.footer-link {:href "http://democracy.works"} " Democracy Works"]]]
```
Comment (tank157) [inline src/cljs/gttp2/views.cljs]: Let's change the Democracy Works link to use `https` and be good web citizens.

Comment (ddellacosta) [reply src/cljs/gttp2/views.cljs]: Good catch!

**Comment by tank157** [comment]
Thanks!

**Pull request merged**